### PR TITLE
fix: undo really trivial styling issue that happened, nothing to see here honest

### PR DESCRIPTION
**Pull request by gashcrumb**
has nothing to do with https://github.com/syndesisio/syndesis/issues/5732

@seanforyou23 shhh

### CODE REVIEW

File: app/ui-react/syndesis/src/index.tsx [hunk 1]
```diff
@@ -3,14 +3,16 @@ import {
   ServerEventsContext,
   WithServerEvents,
 } from '@syndesis/api';
+// Don't move this line, otherwise we anger the pf4
```
Comment (seanforyou23) [inline app/ui-react/syndesis/src/index.tsx]: I should have added a warning like this the first time around, good call!

**Comment by riccardo-forina** [comment]
Wha? I don’t wanna know

**Comment by seanforyou23** [comment]
@riccardo-forina it looks like if we import anything from `@syndesis/ui` before we import the main App `import { App, ... } from './app';` then the order of injected css styles is wrong in the resulting DOM. App needs to load first before other component stylesheets since it is what's importing the base styles (which contain resets).

Could probably refactor how these imports are structured but for now this get's us back to the stable point we've been seeing all week. I know you said you didn't wanna know, but I wanted you to know :)

**Comment by riccardo-forina** [comment]
Now I want to unknown it all.

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/2586/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Pull request merged**